### ISSUE TITLE
conformance: `HTTPRouteDisallowedKind` test

### ISSUE THREAD

**Issue by mlavacca** [Kong/kong-operator#67]
## Problem Statement

The test `HTTPRouteDisallowedKind` is part of the HTTP and TLS conformance. We need to run such a core test successfully. 

## Acceptance Criteria

- [ ] The core test `HTTPRouteDisallowedKind` is successfully run.

**Comment by pmalek** [comment]
This is blocked by #64 

```
=== RUN   TestGatewayConformance/HTTPRouteDisallowedKind
    suite.go:240: Skipping HTTPRouteDisallowedKind: suite does not support TLSRoute
```

**Comment by mlavacca** [comment]
Removing from the milestone, as it requires TLS listeners to be implemented, which is out of scope for 1.3

**Comment by mlavacca** [comment]
Blocked by #64
